File: Backend/TN.TNM.DataAccess/Models/WareHouse/InventoryReceivingVoucherModel.cs
using System;
using System.Collections.Generic;
using System.Text;
using Entities=TN.TNM.DataAccess.Databases.Entities;

namespace TN.TNM.DataAccess.Models.WareHouse
{
    public class InventoryReceivingVoucherModel
    {
        public Guid InventoryReceivingVoucherId { get; set; }
        public string InventoryReceivingVoucherCode { get; set; }
        public Guid StatusId { get; set; }
        public int InventoryReceivingVoucherType { get; set; }
        public Guid WarehouseId { get; set; }
        public string ShiperName { get; set; }
        public Guid? Storekeeper { get; set; }
        public DateTime InventoryReceivingVoucherDate { get; set; }
        public TimeSpan InventoryReceivingVoucherTime { get; set; }
        public int LicenseNumber { get; set; }
        public bool Active { get; set; }
        public DateTime CreatedDate { get; set; }
        public Guid CreatedById { get; set; }

        public Guid VendorId { get; set; }
        public Guid CustomerId { get; set; }
        public string NameStorekeeper { get; set; }
        public string NameVendor { get; set; }
        public string CustomerName { get; set; }
        public string NameCreate { get; set; }
        public string NameStatus { get; set; }

        public string ProducerName { get; set; }
        public int? OrderNumber { get; set; }
        public DateTime? OrderDate { get; set; }
        public decimal? InvoiceNumber { get; set; }
        public DateTime? InvoiceDate { get; set; }
        public int? InventoryReceivingVoucherCategory { get; set; }
        public int? BoxGreen { get; set; }
        public int? BoxGreenMax { get; set; }
        public int? PalletMax { get; set; }
        public int? PalletNormal { get; set; }
        public int? BoxBlue { get; set; }
        public int? PalletSmall { get; set; }

        public List<Entities.VendorOrder> ListVendorOrder { get; set; }
        public List<Entities.CustomerOrder> ListCustomerOrder { get; set; }

    }
}

File: Fontend/TNT.N8.Web/TNT.N8.Web/ClientApp/src/app/warehouse/models/InventoryReceivingVoucher.model.ts
export class InventoryReceivingVoucherModel {
  InventoryReceivingVoucherId: string
  InventoryReceivingVoucherCode: string
  StatusId: string
  Active: boolean
  UpdatedDate: Date
  UpdatedById: string
  Description: string
  Note: string
  ObjectId: string
  Storekeeper: string
  InventoryReceivingVoucherTime: Date
  LicenseNumber: number = 0
  ExpectedDate: Date
  PartnersId: string
  InventoryReceivingVoucherCategory: number

  InventoryReceivingVoucherType: number; // loại phiếu
  InventoryReceivingVoucherTypeName: string; // tên phiếu
  InventoryReceivingVoucherDate: Date; //ngày nhận
  // inventoryReceivingVoucherTime: string;
  PartnersName: string;
  CreatedName: string
  StatusName: string

  // licenseNumber: number;
  OrderDate: Date //ngày đặt hàng
  CreatedDate: Date; // ngày tạo
  CreatedById: string; //người lập phiếu
  WarehouseCategoryTypeId: string //loại kho
  ShiperName: string //giao hàng
  InvoiceNumber: number; //số hóa đơn
  InvoiceDate: Date; // Ngày
  ProducerName: string //Tên nhà sản xuất
  OrderNumber: number
  WarehouseId: string // id kho nhận

  BoxGreen: number
  BoxGreenMax: number
  PalletMax: number
  PalletNormal: number
  BoxBlue: number
  PalletSmall: number

  VendorId: string

  constructor() {
    // this.active = true;
  }
}
